packages/trader/src/Assets/Trading/Categories/accumulator-description-video.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { getUrlBase, isMobile } from '@deriv/shared';
+import { useStore } from '@deriv/stores';
+
+const AccumulatorDescriptionVideo = ({ selected_contract_type }: { selected_contract_type: string }) => {
+    const { ui } = useStore();
+    const { is_dark_mode_on: is_dark_theme } = ui;
+    const [is_loading, setIsLoading] = React.useState(true);
+    const getVideoSource = React.useCallback(
+        (extension: 'mp4' | 'webm') => {
+            return getUrlBase(
+                `/public/images/common/accumulator_description${is_dark_theme ? '_dark' : '_light'}.${extension}`
+            );
+        },
+        [is_dark_theme]
+    );
+
+    // memoize file paths for videos and open the modal only after we get them
+    const mp4_src = React.useMemo(() => getVideoSource('mp4'), [getVideoSource]);
+    const webm_src = React.useMemo(() => getVideoSource('webm'), [getVideoSource]);
+
+    if (selected_contract_type !== 'accumulator') return null;
```

Comment:
There is a case where this component can be shown for non-accumulators contract?

Reply:
At this moment no, will have to change this logic slightly as we add more video.